src/ide/completion/expr/typing.rs

```diff
@@ -0,0 +1,46 @@
+use cairo_lang_defs::ids::{FileIndex, ModuleFileId, NamedLanguageElementId};
+use lsp_types::CompletionItem;
+
+use crate::ide::completion::helpers::completion_kind::importable_completion_kind;
+use crate::lang::analysis_context::AnalysisContext;
+use crate::lang::db::AnalysisDatabase;
+use cairo_lang_semantic::db::SemanticGroup;
+use cairo_lang_syntax::node::TypedSyntaxNode;
+use cairo_lang_syntax::node::ast::ExprPath;
+use cairo_lang_syntax::node::kind::SyntaxKind;
+use if_chain::if_chain;
+
+pub fn typing_completions(db: &AnalysisDatabase, ctx: &AnalysisContext<'_>) -> Vec<CompletionItem> {
```

Comment:
I think it describes better what's actually happening 

```suggestion
pub fn importable_completions(db: &AnalysisDatabase, ctx: &AnalysisContext<'_>) -> Vec<CompletionItem> {
```